Packages/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+	"dependencies": {
```

Comment:
Should we ignore or include this in our project files?

Reply:
Ahh, this slipped past. I'm not sure...it seems like recent Unity versions are generating this, but I don't think we'd need it unless we use it. I can add it to the .gitignore.

Reply:
Yeah, I'm unsure exactly what these dependencies are or what they're used for.